File: app/src/main/java/com/kimjisub/launchpad/adapter/StoreItem.java
package com.kimjisub.launchpad.adapter;

import com.kimjisub.design.PackViewSimple;
import com.kimjisub.launchpad.manager.Unipack;
import com.kimjisub.launchpad.network.fb.fbStore;

public class StoreItem {
	public com.kimjisub.launchpad.network.fb.fbStore fbStore;
	public boolean isDownloaded;
	public boolean isDownloading;

	public PackViewSimple packViewSimple;
	public boolean isToggle = false;

	public StoreItem(fbStore fbStore, boolean isDownloaded) {
		this.fbStore = fbStore;
		this.isDownloaded = isDownloaded;
	}
}

File: app/src/main/java/com/kimjisub/launchpad/adapter/StoreItem.kt
package com.kimjisub.launchpad.adapter

import com.kimjisub.design.PackView
import com.kimjisub.launchpad.network.fb.StoreVO

class StoreItem(
		var storeVO: StoreVO,
		var isDownloaded: Boolean = false
) {
	var isDownloading: Boolean = false


	var packView: PackView? = null
	var isToggle = false

}
